.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+**Screenshots**
+
+**Related Issue**
+E.g: *Closes/Fixes #1234*
+
+- [ ]  **Code Quality**
+    - [ ]  The code follows established coding standards and style guidelines
+    - [ ]  The code is properly commented on and provides context when needed (complex feature or hard to understand)
+- [ ]  **Functionality validation**
+    - [ ]  Feature working as described in the related issue
+    - [ ]  Does this PR need environment variables to be set? If yes, have they been set?
+    - [ ]  New API endpoints are properly documented
+    - [ ]  New functions are properly tested (When should we add tests?)
```

Comment:
I think we could aim to have all service methods tested as the baseline. Routes and jobs on most cases should rely on a service that should be tested.